packages/api/public/spec/paths/translation/verse-glosses.yml

```diff
@@ -61,3 +61,43 @@ get:
                     suggestions: ['diciendo']
                     state: UNAPPROVED
                     machineGloss: 'diciendo'
+patch:
+  tags:
+    - Languages
+  summary: Patch the glosses of a verse for a language.
+  parameters:
+    - $ref: '../../schemas/parameters.yml#languageCode'
+    - $ref: '../../schemas/parameters.yml#verseId'
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          properties:
+            data:
+              type: object
+              additionalProperties:
+                type: object
```

Comment:
This `additionalProperties` isn't quite right. See `localhost:4300/explorer` to see how it shows up.

![image](https://github.com/globalbibletools/gbt/assets/17209828/c9d47015-3b89-4b12-afc9-b6b4bda66596)

I think if you remove these two lines and then dedent the rest of the lines, it should fix it.

Reply:
@Pertempto The only thing is that, at the current moment, data is passed in as a record, with the `wordId` as the key:
```
  data: {
    [wordId: string]: { gloss?: string; state?: GlossState };
  };
```
I think we _could_ fix the YAML file without changing the request type, but do you think it be better to change the request body type and go with the fix you prescribed?

Reply:
We don't change the request type just to match the documentation, you change the documentation to match the request. I guess I don't have enough experience in swagger to give you a good answer on how you'll make the swagger match your request, but there's plenty of good examples and documentation online.

Reply:
Done.

Reply:
Oh shoot, you're right! :rofl: Those were verseIds. I'll fix that.

Reply:
Fixed :+1: